app/javascript/onboarding/components/FollowUsers.jsx

```diff
@@ -176,40 +176,45 @@ class FollowUsers extends Component {
             </header>
 
             <div data-testid="onboarding-users">
-              {users.map((user) => (
-                <button
-                  data-testid="onboarding-user-button"
-                  type="button"
-                  onClick={() => this.handleClick(user)}
-                  onKeyDown={() => this.handleKeyDown(user)}
-                  className={
-                    selectedUsers.includes(user)
-                      ? 'user content-row selected'
-                      : 'user content-row unselected'
-                  }
-                >
-                  <figure className="user-avatar-container">
-                    <img
-                      className="user-avatar"
-                      src={user.profile_image_url}
-                      alt="profile"
-                    />
-                  </figure>
-                  <div className="user-info">
-                    <h4 className="user-name">{user.name}</h4>
-                    <p className="user-summary">
-                      {he.unescape(user.summary || '')}
-                    </p>
-                  </div>
+              {users.map((user) => {
+                const selected = selectedUsers.includes(user);
+
+                return (
                   <button
-                    data-testid="onboarding-user-following-status"
+                    data-testid="onboarding-user-button"
                     type="button"
-                    className="user-following-status"
+                    onClick={() => this.handleClick(user)}
+                    className={
+                      selected
+                        ? 'user content-row selected'
+                        : 'user content-row unselected'
+                    }
                   >
-                    {selectedUsers.includes(user) ? 'Following' : 'Follow'}
+                    <figure className="user-avatar-container">
+                      <img
+                        className="user-avatar"
+                        src={user.profile_image_url}
+                        alt="profile"
+                      />
+                    </figure>
+                    <div className="user-info">
+                      <h4 className="user-name">{user.name}</h4>
+                      <p className="user-summary">
+                        {he.unescape(user.summary || '')}
+                      </p>
+                    </div>
+                    <button
+                      data-testid="onboarding-user-following-status"
+                      type="button"
+                      className="user-following-status"
+                      aria-pressed={selected}
+                      aria-label={`Follow ${user.name}`}
+                    >
+                      {selected ? 'Following' : 'Follow'}
+                    </button>
                   </button>
-                </button>
```

Comment:
Refactor (blocking): the accessibility issues here extend slightly beyond the attributes you've added, as we have a `button` as a descendant of another `button`. At the moment, testing with VoiceOver on Safari, the useful `aria` you've added is unfortunately being ignored because of this.

I think this fix will need to include turning the containing button (line 183) into a `div`. We can leave the click handler on that div so that mouse users can still click the whole row (some of our eslint a11y rules will need disabled for that line, which in this case should be fine given we have the keyboard accessible `button` at line 206).

This should make sure that your changes are picked up and announced by the screen reader 🙂  Any questions on this, just give me a shout 👍